### DISCUSSION TITLE
How to integrate react-spring with requestAnimationFrame API?

### DISCUSSION THREAD

**Discussion by jeremygottfried** [pmndrs/react-spring#1994] [Support]
**Background**
I've always used the FLIP animation technique (first last invert play), which relies heavily on `requestAnimationFrame`. More info - https://aerotwist.com/blog/flip-your-animations/

**Question**
The answer to this question might seem obvious to others, but the docs don't seem to mention `requestAnimationFrame` at all. I was wondering if react-spring automatically uses `requestAnimationFrame`? If not, how would I integrate it with `useSpring`?

**Comment by joshuaellis** [comment]
The library uses RAF. We've created a abstraction class around it called rafz.

**Comment by AnweshGangula** [comment]
@jeremygottfried, did you find any useful resources for this?
